test/common/deployment_types.go

```diff
@@ -2,6 +2,13 @@ package common
 
 import (
 	goctx "context"
+	"github.com/integr8ly/integreatly-operator/pkg/resources/quota"
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	"golang.org/x/net/context"
+	v12 "k8s.io/api/apps/v1"
+	v13 "k8s.io/api/core/v1"
```

Comment:
Can you rename these imports please?
`v13` -> `corev1`
and `v12` to something like `k8sappsv1` ?